<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlotterWindow</class>
 <widget class="QMainWindow" name="PlotterWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1121</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QPlotter</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/image/icon.svg</normaloff>:/image/icon.svg</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>2</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="functionLabel">
           <property name="font">
            <font>
             <family>Courier New</family>
            </font>
           </property>
           <property name="text">
            <string>f(x)=</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="functionInput"/>
         </item>
         <item>
          <widget class="QPushButton" name="plotButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/image/icon.svg</normaloff>:/image/icon.svg</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QListWidget" name="historyList">
         <property name="font">
          <font>
           <family>Courier New</family>
          </font>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="PlotWidget" name="plotCanvas" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>4</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>400</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>400</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: white;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>2</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QTableWidget" name="variableTable">
         <property name="font">
          <font>
           <family>Courier New</family>
          </font>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Variable</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Wert</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1121</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header>plotwidget.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
